src/main/java/com/solr/clientwrapper/rest/EmployeeController.java

```diff
@@ -0,0 +1,129 @@
+package com.solr.clientwrapper.rest;
+
+
+import com.solr.clientwrapper.infrastructure.entity.Employee;
+import com.solr.clientwrapper.infrastructure.repository.EmployeeRepository;
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.client.solrj.response.FacetField;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.apache.solr.common.SolrDocumentList;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+public class EmployeeController {
```

Comment:
This class was removed. Why did you add this again

Reply:
i was merge that project with my pervious repository